presto-parquet/src/main/java/io/prestosql/parquet/writer/PrimitiveColumnWriter.java

```diff
@@ -296,7 +296,10 @@ private void flushCurrentPageToBuffer()
     @Override
     public long getBufferedBytes()
     {
-        return pageBuffer.stream().mapToLong(ParquetDataOutput::size).sum() + definitionLevelEncoder.getBufferedSize() + repetitionLevelEncoder.getBufferedSize();
+        return pageBuffer.stream().mapToLong(ParquetDataOutput::size).sum() +
```

Comment:
I would squash this into the commit that added getBufferedBytes.  I think it was `Expose written bytes and buffered bytes in ParquetWriter `